presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -121,6 +125,53 @@ public void testPublic()
         assertThat(aliceExecutor.executeQuery(format("SELECT * FROM %s", tableName))).hasNoRows();
     }
 
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testTableOwnerPrivileges()
+    {
+        String tableName = "alice_table";
+        QueryExecutor aliceExecutor = connectToPresto("alice@presto");
+
+        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+
+        onHive().executeQuery("set role admin;");
```

Comment:
I'm surprised that this persists between executeQuery statements, because I thought that they each had a separate connection, but as long as this works it's fine